### PR TITLE
Fix #1374 :Add IME Action Done 

### LINKED ISSUE #1374: Add IME Action Done
We should add Action Done here on the keyboard, as user want to create a profile just after entering his/her name. 
Or we also have an option to create 3 digit pin here on the screen, so a user might want to do that after entering his/her name, so we can also close the soft keyboard on the action done. 
Providing a new line in the `name` edit text might cause some issues and this also not required. 

### How to produce this screen? 
- Click on new add new profile from the profile chooser 
<br>

Screenshot_1592807590

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix #1374 
Set single line true for name input in add profile screen

Screenshot_1594015269

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/add_profile_activity.xml [hunk 1]
```diff
@@ -85,7 +85,8 @@
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_required_heading_text_view"
-            profile:error="@{viewModel.nameErrorMsg}" />
+            profile:error="@{viewModel.nameErrorMsg}"
+            profile:singleLine="@{true}" />
```
Comment (rt4914) [inline app/src/main/res/layout/add_profile_activity.xml:89]: This same change should be done in `land/add_profile_activity.xml` too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Change suggested

File: app/src/main/res/layout/add_profile_activity.xml (hunk 1 above)
Comment (anandwana001) [reply app/src/main/res/layout/add_profile_activity.xml:89]: Done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**